ssa/go.mod

```diff
@@ -2,11 +2,16 @@ module github.com/fluxcd/pkg/ssa
 
 go 1.18
 
+// Temporary replace directive to ensure compatibility with k8s 1.27
+// Ref: https://github.com/kubernetes-sigs/controller-runtime/pull/2189
+replace sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.13.1-0.20230504051641-57674c8dd62f
```

Comment:
This will cause major issues downstream as every consumer must also use a replacement. Please set the controller-runtime to v0.13.1-0.20230504051641-57674c8dd62f in the require section.